hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/Constants.java

```diff
@@ -1252,6 +1253,33 @@ private Constants() {
   public static final String AWS_SERVICE_IDENTIFIER_DDB = "DDB";
   public static final String AWS_SERVICE_IDENTIFIER_STS = "STS";
 
+  /** Prefix for S3A client-specific properties.
+   * value: {@value}
+   */
+  public static final String FS_S3A_CLIENT_PREFIX = "fs.s3a.client.";
+
+  /** Custom headers postfix.
+   * value: {@value}
+   */
+  public static final String CUSTOM_HEADERS_POSTFIX = ".custom.headers";
+
+  /**
+   * List of custom headers to be set on the service client.
+   * Multiple parameters can be used to specify custom headers.
```

Comment:
`value: {@value}` wouldn't render with toLowerCase(), added for other statements.

Reply:
ok. I'm going to propose here doing all the next lines
in a single &lt;pre> as there's enough formatting it wouldn't render right. you can then cut the {@literal} bits.

The goal here is generally "looks good on an IDE rendered view"